py/desispec/workflow/tableio.py

```diff
@@ -0,0 +1,304 @@
+import os
+import numpy as np
+from astropy.table import Table
```

Comment:
See earlier comment about maintainability of these table structures.  The sheer amount of custom code here for reading/writing/parsing these tables is a maintainability red flag.  If you find yourself having to make any more updates here to get basic functionality working, let's pause and reconsider whether further updates or restructuring is the right path.

Reply:
The code is required for users to be able to include multiple comments into the exposure tables. That code can then be utilized by the other multi-value cells like ALL_QIDS, DEP_IDS, etc.

The code here is also with regards to guessing default values and overcoming masked columns. Those things are not changed by having scalar-valued elements.

Agreed that it is something to be revisited regularly, especially if bugs/issues/backward compatibility issues start to arise.